src/TextArea/TextArea.less

```diff
@@ -0,0 +1,98 @@
+@import (reference) "../less/colors";
+@import (reference) "../less/border";
+@import (reference) "../less/sizing";
+@import (reference) "../less/type-size";
+@import (reference) "../less/type-utilities";
+@import (reference) "../less/spacing";
+
+.TextArea {
+  .padding--left--s;
```

Comment:
Removes awkward whitespace on the right when a textbox is vertically but not horizontally resizable (the resize icon is on the bottom right).